test/input_files/failed/failed_fileformat_000.vcf

```diff
@@ -1,4 +1,4 @@
-##fileformat=
+##qformat=
```

Comment:
I was just testing how it failed, I forgot to restore this file. I will restore it now.